index.js

```diff
@@ -0,0 +1,12 @@
+const { parse, isValid } = require('./lib/input.js');
+const { execute } = require('./lib/notes.js');
+
+let parsedArray = parse(process.argv);
+let response;
+
+if(isValid(parsedArray)) {
+    response = execute(parsedArray);
+}
+
+console.log(response);
+console.log(parsedArray)
```

Comment:
files should end with a newline. you can configure vs code to add it for you automatically